lang/lang_en_it.txt

```diff
@@ -1241,3 +1241,195 @@
 #MSG_OFF
 " [off]"
 "\x00"
+
+#
+"Back"
+"Indietro"
+
+#
+"Checks"
+"Controlli"
+
+#
+"False triggering"
+"Falso innesco"
+
+#
+"FINDA:"
+"\x00"
+
+#
+"Firmware   [none]"
+"Firmware[nessuno]"
+
+#
+"Firmware [strict]"
+"Firmware [esatto]"
+
+#
+"Firmware   [warn]"
+"Firmware [avviso]"
+
+#
+"HW Setup"
+"Installazione HW"
+
+#
+"IR:"
+"\x00"
+
+#
+"Magnets comp.[N/A]"
+"Comp. Magneti[N/A]"
+
+#
+"Magnets comp.[Off]"
+"Comp. Magneti[off]"
+
+#
+"Magnets comp. [On]"
+"Comp. Magneti [on]"
+
+#
+"Mesh         [3x3]"
+"Griglia      [3x3]"
+
+#
+"Mesh         [7x7]"
+"Griglia      [7x7]"
+
+#
+"Mesh bed leveling"
+"Mesh livel. letto"
+
+#
+"MK3S firmware detected on MK3 printer"
+"Firmware MK3S rilevato su stampante MK3"
+
+#
+"MMU Mode [Normal]"
+"Modalita MMU [Normale]"
+
+#
+"MMU Mode[Stealth]"
+"Modalita MMU [Silenziosa]"
+
+#
+"Mode change in progress ..."
+"Cambio modalita in corso ..."
+
+#
+"Model      [none]"
+"Modello [nessuno]"
+
+#
+"Model    [strict]"
+"Modello  [esatto]"
+
+#
+"Model      [warn]"
+"Modello  [avviso]"
+
+#
+"Nozzle d.  [0.25]"
+"Diam.Ugello[0.25]"
+
+#
+"Nozzle d.  [0.40]"
+"Diam.Ugello[0.40]"
+
+#
+"Nozzle d.  [0.60]"
+"Diam.Ugello[0.60]"
+
+#
+"Nozzle     [none]"
+"Ugello  [nessuno]"
+
+#
+"Nozzle   [strict]"
+"Ugello   [esatto]"
+
+#
+"Nozzle     [warn]"
+"Ugello   [avviso]"
+
+#
+"G-code sliced for a different level. Continue?"
+"G-code processato per un livello diverso. Continuare?"
+
+#
+"G-code sliced for a different level. Please re-slice the model again. Print cancelled."
+"G-code processato per un livello diverso. Per favore esegui nuovamente lo slice del modello. Stampa annullata."
+
+#
+"G-code sliced for a different printer type. Continue?"
+"G-code processato per una stampante diversa. Continuare?"
+
+#
+"G-code sliced for a different printer type. Please re-slice the model again. Print cancelled."
+"G-code processato per una stampante diversa. Per favore esegui nuovamente lo slice del modello. Stampa annullata."
+
+#
+"G-code sliced for a newer firmware. Continue?"
+"G-code processato per un firmware piu recente. Continuare?"
+
+#
+"G-code sliced for a newer firmware. Please update the firmware. Print cancelled."
+"G-code processato per un firmware piu recente. Per favore aggiorna il firmware. Stampa annullata."
+
+#
+"PINDA:"
+"\x00"
+
+#
+"Preheating to cut"
+"Preriscaldamento per taglio"
+
+#
+"Preheating to eject"
+"Preriscaldamento per espulsione"
+
+#
+"Printer nozzle diameter differs from the G-code. Continue?"
+"Diametro ugello diverso da G-Code. Continuare?"
+
+#
+"Printer nozzle diameter differs from the G-code. Please check the value in settings. Print cancelled."
+"Diametro ugello diverso dal G-Code. Controlla il valore nelle impostazioni. Stampa annullata."
+
+#
+"Rename"
+"Rinomina"
+
+#
+"Select"
+"Seleziona"
+
+#
+"Sensor info"
+"Info Sensore"
+
+#
+"Sheet"
+"Piano"
+
+#
+"Sound     [assist]"
```

Comment:
After following https://github.com/DRracer/Prusa-Firmware/blob/translations-doc/lang/translations.md the not_tran.txt showed that Sound [assist] is missing.
Checking the lang_en.txt and lang_en_??.txt it seams to be a typo using one space too much.

The message should be 17 chars long and not 18 with 4 spaces between Sound and [assist]